static/js/interactive-map/PinClusterer/PinClusterer.js

```diff
@@ -205,6 +295,13 @@ class PinClusterer {
     }
   }
 
+  /**
+   * @returns {PinCluster[]} Current pin clusters
+   */
+  getClusters() {
+    return [...this._clusters];
```

Comment:
Is there a reason for the spread operator here rather than just returning this._clusters?

Reply:
Not sure! It might be so that the array can be changed, but I know it doesn't stop the inner Pin Cluster object from being changed.

Deferring to SLAP-1094!